src/Controller/StatusController.php

```diff
@@ -31,6 +41,57 @@ public function statusAction(Request $request)
         return $this->render('@Clifton/Status/status.html.twig', [
             'now' => new DateTime(),
             'dbConnectivity' => $dbalConnection->isConnected() || $dbalConnection->connect(),
+            'dbErrorCheck' => $hasDBIssues,
```

Comment:
Our other check is 'true when things are good', while this one is 'true when things are bad'. We should pick one convention and stick with it.